age_calculator/age_calculator.py

```diff
@@ -10,4 +9,6 @@ def age_in_days(date_of_birth, date_of_death=None):
 
 def age_in_hours(date_of_birth, date_of_death=None):
     """Calculates the age in hours."""
-    raise NotImplemented
+    date_of_death = arrow.get(date_of_death,'US/Pacific') if date_of_death else arrow.utcnow()
```

Comment:
Note that you are duplicating code here, line #7 contains the same code. (I would suggest to stick to the default locale, so leave out 'US/Pacific'